demos/imagenet_classification_demo/main.cpp

```diff
@@ -397,25 +410,22 @@ int main(int argc, char *argv[]) {
                     if (predictionResult != PredictionResult::Correct) {
                         predictedLabels.push_back(labels[results[i][0]]);
                     }
-                    totalPredictionsCount++;
 
                     shownImagesInfo.push_back(
                         std::make_tuple(completedInferRequestInfo.images[i].mat, predictedLabels[i], predictionResult));
                 }
 
                 framesNum += FLAGS_b;
-
-                typedef std::chrono::duration<double, std::chrono::seconds::period> Sec;
-                avgFPS = 1. * framesNum / std::chrono::duration_cast<Sec>(
+                
+                avgFPS = framesNum / std::chrono::duration_cast<Sec>(
                     std::chrono::steady_clock::now() - startTickCount).count();
                 gridMat.updateMat(shownImagesInfo);
                 auto processingEndTime = std::chrono::steady_clock::now();
-                for (size_t i = 0; i < FLAGS_b; i++) {
-                    latencySum += (1. * std::chrono::duration_cast<Sec>(
-                        processingEndTime - completedInferRequestInfo.images[i].startTime).count());
+                for (const auto & image : completedInferRequestInfo.images) {
+                    latencySum += (1. * std::chrono::duration_cast<Sec>(processingEndTime - image.startTime).count());
```

Comment:
`latencySum` can be of type `std::chrono::steady_clock::duration`. The value can be cased later. `1. * ` is not needed.

Reply:
Brackets are not needed